### PR TITLE
Extension

### PR DESCRIPTION
**Pull request by AdaRoseCannon**
Add a simple extension which wraps around the existing bookmarklet.

### CODE REVIEW

File: extension/manifest.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "description": "A Chrome Extension that automagically adds a performance analysis widget.",
+  "version": "1.0.0",
+
+  "browser_action": {
+    "default_icon": "icon.png",
+    "default_popup": "dialog.html"
+  },
+  "content_scripts": [
+    {
+      "matches": ["*://*.ft.com/*"],
```
Comment (JakeChampion) [inline extension/manifest.json:14]: :+1:

File: extension/scripts/main.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+'use strict';
+/*global chrome*/
+
+chrome.runtime.sendMessage({method: 'isEnabled'}, function(response) {
+	if (!response.enabled) return;
+	window.perfWidgetUrl = 'https://ftlabs-perf-widget-test.herokuapp.com';
```
Comment (JakeChampion) [inline extension/scripts/main.js]: Could this be configurable somehow?

File: extension/manifest.json (hunk 1 above)
Comment (seanmtracey) [inline extension/manifest.json:14]: :+1:

File: extension/scripts/background.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+chrome.runtime.onMessage.addListener(function(request, sender, sendResponse) {
+
+	if (request.method === 'isEnabled') {
+		sendResponse({enabled});
+	}
+
+	if (request.method === 'setEnabled') {
+		enabled = request.enabled;
+		localStorage.setItem('enabled', Number(request.enabled));
```
Comment (JakeChampion) [inline extension/scripts/background.js]: :+1:

File: extension/scripts/main.js (hunk 2 above)
Comment (seanmtracey) [inline extension/scripts/main.js]: It could be set as part of the gulp build, like the bookmarklet

File: package.json [hunk 4]
```diff
@@ -48,6 +48,7 @@
   "devDependencies": {
     "chai": "^3.4.1",
     "chai-as-promised": "^5.1.0",
+    "gulp-webpack": "^1.5.0",
```
Comment (JakeChampion) [inline package.json]: Needs to be in the other dep section as we run the build script on Heroku and Heroku does a `npm i --production`

Comment (AdaRoseCannon) [inline package.json]: Ah thanks Jake :)

**Comment by JakeChampion** [comment]
LGTM :+1:

**Pull request merged**